CodeEditModules/Modules/StatusBar/src/Model/StatusBarModel.swift

```diff
@@ -39,17 +51,25 @@ public class StatusBarModel: ObservableObject {
     public var currentCol: Int = 1 // Implementation missing
 
     /// Returns true when the drawer is visible
-    @Published
+    @AppStorage("statusbar.isExpanded")
```

Comment:
Use an `enum` for UserDefaults keys instead of string literals to minimize error chances by mistyping.

Reply:
Also this will expand collapse all status bar drawers across multiple open workspaces.